cmd/commands/update_index.go

```diff
@@ -43,7 +43,7 @@ var UpdateIndexCmd = &cobra.Command{
 }
 
 func getLongUpdateDescription() string {
-	return `Updates the public index in ` + os.Getenv("CMSIS_PACK_ROOT") + `/.Web/index.pidx using the URL in <url> tag inside index.pidx.
+	return `Updates the public index in ` + os.Getenv("CMSIS_PACK_ROOT") + "/.web/" + installer.PublicIndex + " using the URL in <url> tag inside " + installer.PublicIndex + `.
```

Comment:
Was passiert hier, wenn CMSIS_PACK_ROOT nicht im User Environment gesetzt ist. Setzt cpackget das auf default?

Reply:
Da ist dann nichts. Muss ich wohl vorher abfragen. Aber ist nur für die Help-Message...

Reply:
If CMSIS_PACK_ROOT is not set, than cpackget uses the host specific default locations like e.g. on windows:
"c:\users\<user>\AppData\Local\Arm\Packs" or if `-R <dir>` is specified then that location is used. 
I am pretty certain that there must be a variable representing this location for the running session.